pyccel/parser/syntactic.py

```diff
@@ -385,191 +340,256 @@ def _visit_NoneType(self, stmt):
 
     def _visit_str(self, stmt):
 
-        return repr(stmt)
+        return stmt
 
-    def _visit_StringNode(self, stmt):
-        val =  stmt.value
-        if isinstance(stmt.parent,(RedBaron, DefNode)):
+    def _visit_Str(self, stmt):
+        val =  stmt.s
+        if isinstance(self._scope[-2], ast.Expr):
             return CommentBlock(val)
```

Comment:
why don't we use `get_docstring` function from the `ast` module ?
see here https://docs.python.org/3.5/library/ast.html#ast.get_docstring

Reply:
I didn't realise this function existed. Does it still work if there is more than one comment block? (Ie if a docstring is used as a comment block rather than as an actual doc string?

Reply:
it won't work , I think this way is better